cmd/oras/root/pull.go

```diff
@@ -137,11 +137,15 @@ func runPull(ctx context.Context, opts pullOptions) error {
 		}
 		return err
 	}
+
+	// suggest oras copy for pulling layers without annotation
 	if pulledEmpty {
-		fmt.Println("Downloaded empty artifact")
+		fmt.Printf("Skipped pulling layers without %q\n", ocispec.AnnotationTitle)
+		fmt.Printf("Use 'oras copy %s --to-oci-layout <layout-dir>' to pull all layers.\n", opts.RawReference)
```

Comment:
Currently `pulledEmpty == true` means an artifact **only** contains unnamed layers. What if an artifact contains both named and unnamed layers? Should we also output this error message?

Reply:
I would revisit it when those start happening right now the error is confusing. What is an empty artifact from the point of CLI. The variable name should also be something like skippedLayer.

Reply:
Ok fixed the variable name to count the skipped layers. 

<img width="581" alt="image" src="https://github.com/oras-project/oras/assets/1821104/d214755f-a631-4ee0-868c-469cd48434a6">

The manifest had partial number of layers 

```json
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
   "config": {
      "mediaType": "application/vnd.docker.container.image.v1+json",
      "size": 1470,
      "digest": "sha256:9c7a54a9a43cca047013b82af109fe963fde787f63f9e016fdc3384500c2823d"
   },
   "layers": [
      {
         "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
         "size": 2457,
         "digest": "sha256:719385e32844401d57ecfd3eacab360bf551a1491c05b85806ed8f1b08d792f6",
         "annotations" : {
                "org.opencontainers.image.title" : "thelayer"
            }
      }
   ]
}
```